node/src/chain_spec.rs

```diff
@@ -88,21 +114,353 @@ pub fn mangata_inflation_config() -> InflationInfo<Balance> {
 	}
 }
 
+pub fn kusama_mainnet_config() -> ChainSpec {
+	// Give your base currency a unit name and decimal places
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), "MGX".into());
+	properties.insert("tokenDecimals".into(), 18.into());
+	properties.insert("ss58Format".into(), 42.into());
+
+	ChainSpec::from_genesis(
+		// Name
+		"Mangata Kusama Mainnet",
+		// ID
+		"mangata_kusama_mainnet",
+		ChainType::Development,
```

Comment:
shouldnt that be `Local` or `Live` ? 
https://github.com/mangata-finance/substrate/blob/mangata-dev/primitives/chain-spec/src/lib.rs#L26-L33

Reply:
It should be Live

Reply:
Right! Changed.
Thanks for noticing that.